Formula/stolon.rb

```diff
@@ -0,0 +1,33 @@
+$version = "v0.13.0"
+$bin_name = "stolon"
+
+class Stolon < Formula
+    desc "Stolon is a cloud native PostgreSQL manager for high availability"
+    homepage "https://github.com/sorintlab/stolon"
+    url "https://github.com/sorintlab/#{$bin_name}.git", tag: $version
+    version $version
```

Comment:
this isn't needed, the version should be picked up from the tag

Reply:
Removed the version